frb/dm_kde/transient_data/psrcat.csv

```diff
@@ -0,0 +1,2705 @@
+#,NAME,dm,P0,RAJD,DECJD
```

Comment:
Is all this data in https://github.com/FRBs/pulsars
already?  I suspect so.  If so, remove this file and
just link your code to that other Repo.

Reply:
I've added a pull request to add the file to the repo (v1.61)